packages/headless/src/features/pagination/pagination-selectors.ts

```diff
@@ -1,4 +1,3 @@
-import {createSelector} from '@reduxjs/toolkit';
```

Comment:
I adjusted the selectors in this file to no longer use the `createSelector` utility. The utility caches the inputs to the function, which makes it harder to unit test different inputs.

The reason this problem appears now is because [destructuring a core pager object](https://github.com/coveo/ui-kit/pull/1101/files#diff-19560fb69917a92e382a82c7948d49196ea8dafbc269b614feb556655608c8b4R149) with a `state` getter immediately evaluates the properties of the getter, setting values in cache, and changing the state of the engine in the unit tests does not reset the cache 🤷

Reply:
> The utility caches the inputs to the function, which makes it harder to unit test different inputs.

You can do `const result = selector.resultFunc(inputA, inputB)` to test the different inputs with `createSelector` :smile:

This is what we used to do in admin-ui, at the very least